utilities/e2e-template-assets/module-specific/azure-stack-hci/azureStackHCIClusterPreqs/ashciPrereqs.bicep

```diff
@@ -44,23 +44,6 @@ var arbDeploymentServicePrincipalValue = base64('${arbDeploymentAppId}:${arbDepl
 
```

Comment:
Happy to approve this one - alongside disabling Actions upon merge to avoid a tsumami of triggered workflows